transports/azure-service-bus-netstandard/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Azure Service Bus .Net Standard Transport
+component: ASBS
+tags:
+ - Azure
+reviewed: 2018-06-19
+---
+
+## Azure Service Bus .Net Standard Transport
+
+The Azure Service Bus .Net Standard transport, ASBS in short, is the successor to the [Azure Service Bus transport](transports/azure-service-bus/). It leverages the .Net Standard compatible [Microsoft.Azure.ServiceBus](https://www.nuget.org/packages/Microsoft.Azure.ServiceBus/) client SDK.
+
+[Azure Service Bus (ASB)](https://azure.microsoft.com/en-us/services/service-bus/) is a messaging service hosted on the Azure platform, that allows for exchanging messages between various applications in a loosely coupled fashion. ASB Queues offer <a href="https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics)">"First In, First Out" (FIFO)</a> guaranteed message delivery, and support a range of standard protocols (REST, AMQP, WS*) and APIs (to put messages on and pull messages off the queue). ASB Topics deliver messages to multiple subscribers and facilitate use of the fan-out pattern to deliver messages to downstream systems.
```

Comment:
> ASB Queues offer guaranteed message delivery

Is this necessary? NSB doesn't allow message ordering, so why would we mention at all native feature we don't use?

Reply:
It's the same text we have on ASB doco explaininig what SB is. I don't read guaranteed ordering anywhere. it says guaranteed delivery, which is very important to the nsb message

Reply:
@yvesgoeleven I don't mind that, but FIFO reference has to go 🙂